recipes/docarray/meta.yaml

```diff
@@ -1,71 +0,0 @@
-{% set name = "docarray" %}
```

Comment:
This should not be deleted in this PR, rather rebase to the latest master if you want a clean recipes directory

Reply:
This was from one of my previous forks of the main branch. It was not among the packages that were lined up for feedstock creation. So, when git checks for diffs, this should not cause any problem.

- main > my-own-branch > added-new-recipe > removed new-recipe > added current recipes.

Reply:
Just rebased with upstream. Thank you.